bindings/java/src/org/sleuthkit/datamodel/IngestModuleInfo.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Sleuth Kit Data Model
+ *
+ * Copyright 2011-2016 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *	 http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.datamodel;
+
+/**
+ * Class representing information about an ingest module, used in ingest job
+ * info to show which ingest modules were run.
+ */
+public final class IngestModuleInfo {
+
+	/**
+	 * Used to keep track of the module types
+	 */
+	public static enum IngestModuleType {
+		//DO NOT CHANGE ORDER
```

Comment:
I discussed this with Brian, let's go ahead and provide localizable display names for each member of the enum. That way we will be set if we ever proceed with additional localization for TSK.